aggregator/src/main/scala/weco/concepts/aggregator/ConceptExtractor.scala

```diff
@@ -0,0 +1,109 @@
+package weco.concepts.aggregator
+import grizzled.slf4j.Logging
+import ujson.Obj
+import ujson.Value.Value
+import weco.concepts.common.model._
+import weco.concepts.common.json.JsonOps._
+
+import scala.annotation.tailrec
+
+object ConceptExtractor {
+  val conceptTypes =
+    Seq("Concept", "Person", "Organisation", "Meeting", "Period")
```

Comment:
I wonder if these should come from elsewhere more tied to the model at some point - not an issue for now

Reply:
I agree. It's here because this is the only place where it is currently used, but there is definitely scope for it to be  more model-adjacent.